tests/test_core/test_bbox/test_structures/test_box_converters.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from math import sqrt
+from unittest import TestCase
+
+import numpy as np
+import torch
+from mmdet.structures.bbox import HorizontalBoxes
+from mmengine.testing import assert_allclose
+
+from mmrotate.core.bbox.structures import QuadriBoxes, RotatedBoxes
+
+
+class TestboxModeConverters(TestCase):
+
+    def test_hbox2rbox(self):
+        th_hboxes = torch.Tensor([10, 10, 20, 20]).reshape(1, 1, 4)
```

Comment:
What's the meaning of the `th_` prefix?

Reply:
th means torch. I want to represent it's a Tensor type boxes.